lib/jetstream/api/stream.ex

```diff
@@ -304,6 +304,25 @@ defmodule Jetstream.API.Stream do
     end
   end
 
+  @doc """
+  Purges all of data in the stream but doesn't delete the stream.
+
+  ## Examples
+
+      iex> Jetstream.API.Stream.create(:gnat, %Jetstream.API.Stream{name: "stream", subjects: ["subject"]})
+      iex> Jetstream.API.Stream.purge(:gnat, "stream")
+      :ok
+
+      iex> {:error, %{"code" => 404, "description" => "stream not found"}} = Jetstream.API.Stream.purge(:gnat, "wrong_stream")
```

Comment:
It looks weird to pattern match in an example. I would make it to look as an output like the `:ok` above